python/ray/serve/batching.py

```diff
@@ -130,34 +136,64 @@ async def wait_for_batch(self) -> List[Any]:
 
         return batch
 
-    async def _handle_batches(self, func):
+    def _validate_results(self, results, input_batch_length):
```

Comment:
please add type annotations to all functions in this PR

Reply:
I added type annotations.